assets/app/view/game/part/track.rb

```diff
@@ -29,58 +29,37 @@ def render
           # Array<Array<Path>>
           @routes_paths = @routes.map { |route| route.paths_for(@tile.paths) }
 
-          if @tile.offboards.any?
-            @tile.paths.select(&:offboard).map do |path|
+          tracks = []
+          @tile.paths.select { |path| path.edges.size == 2 }
+          .map { |path| [path, index_for(path)] }
+          .sort_by { |_, index| index || -1 }
+          .map do |path, index|
```

Comment:
map returns an array, you shouldn't be doing tracks << x

tracks = x.map...

Reply:
Gotcha.